reana_client/cli/files.py

```diff
@@ -506,30 +506,43 @@ def move_files(ctx, source, target, workflow, access_token):  # noqa: D301
 @check_connection
 @add_access_token_options
 @click.option("-s", "--summarize", is_flag=True, help="Display total.")
+@click.option(
+    "--filter",
+    "filters",
+    multiple=True,
+    help="Filter results to show only file sizes that match certain filtering "
+    "criteria such as file name or size."
+    "Use `--filter <columm_name>=<column_value>` pairs. "
+    "Available filters are `name` and `size`.",
+)
 @human_readable_or_raw_option
 @click.pass_context
 def workflow_disk_usage(
-    ctx, workflow, access_token, summarize, human_readable_or_raw
+    ctx, workflow, access_token, summarize, filters, human_readable_or_raw
 ):  # noqa: D301
     """Get workspace disk usage.
 
     The `du` command allows to chech the disk usage of given workspace.
 
     Examples: \n
     \t $ reana-client du -w myanalysis.42 -s \n
-    \t $ reana-client du -w myanalysis.42 -s --human-readable
+    \t $ reana-client du -w myanalysis.42 -s --human-readable \n
+    \t $ reana-client du -w myanalysis.42 --filter name=workflow
     """
     from reana_client.api.client import get_workflow_disk_usage
 
     logging.debug("command: {}".format(ctx.command_path.replace(" ", ".")))
     for p in ctx.params:
         logging.debug("{param}: {value}".format(param=p, value=ctx.params[p]))
 
+    search_filter = None
```

Comment:
Very minor: couldn't this default value be set in `click` argument definition?

Reply:
`filters` remain None by default here. Here `search_filter` is the None since I am initiating the variable for referencing in the code if `filters` is not present.
- In case when there is no `filters` and It remains `None`
- In case when filters is present and we do not have any `search_filters` for headers `name` and `size`